symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py

```diff
@@ -0,0 +1,39 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless)
+
+# Description
+## The optical path length between two points of the medium is the distance to which light (optical radiation) would propagate in a vacuum during its passage between these points.

```

Comment:
Between what points?
If we take 2 points, in vacuum it will be straight line. In medium it will also be the same straight line. If there are two different media, line will be broken, and longer than the straight line obviously.